.github/workflows/integration-tests.yml

```diff
@@ -10,6 +10,9 @@ on:
 jobs:
   run-tests:
     runs-on: ubuntu-latest
+    if: github.event_name == 'push' && (github.event.ref == 'refs/heads/main' || github.event.ref == 'refs/heads/dev')
```

Comment:
I think this is not necessary because it's already specify in the workflow level.
<img width="290" alt="image" src="https://github.com/linode/ansible_linode/assets/121905282/d52fab2e-b421-4b17-96fe-65fe124188a8">